arch/arm/src/stm32/stm32f20xxf40xx_flash.c

```diff
@@ -77,12 +78,29 @@
 #define FLASH_OPTKEY2      0x4c5d6e7f
 #define FLASH_ERASEDVALUE  0xff
 
+#if CONFIG_STM32_PROGMEM_STARTBLOCK < 0 || \
```

Comment:
This would help me this a non breaking change.
```suggestion
#if !defined(CONFIG_STM32_PROGMEM_STARTBLOCK)
#  define CONFIG_STM32_PROGMEM_STARTBLOCK 0
#endif

#if !defined(CONFIG_STM32_PROGMEM_LASTBLOCK)
#  define CONFIG_STM32_PROGMEM_LASTBLOCK STM32_FLASH_NPAGES-1
#endif

#if CONFIG_STM32_PROGMEM_STARTBLOCK < 0 || \
```